config/200-dummy-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Can we add some comments which explains why we need the dummy secret?
(# This is a dummy secret, which will be mounted in the controller. It should be patched with credentials of a Service
# Account that can manipulate the GCP resources users are interested in.
# Data is set to dummy --> ZHVtbXkK)